guvnor-webapp/src/main/java/org/guvnor/client/perspectives/M2RepoPerspective.java

```diff
@@ -20,13 +20,13 @@
 import javax.enterprise.event.Event;
```

Comment:
At the very least you should expose ```Presenter.getView()``` and then on the View interface have an ```addHideHandler```.. either way you don't want to create a new View instance to attach the ```HideHandler```.